client/core/core.go

```diff
@@ -508,23 +581,31 @@ func (c *Core) Register(form *Registration) (error, <-chan error) {
 		// Don't abandon registration. The fee is already paid.
 	}
 
-	trigger := coinConfirmationTrigger(coin, uint32(dc.cfg.RegFeeConfirms))
+	trigger := func() (bool, error) {
+		confs, err := coin.Confirmations()
+		if err != nil {
+			return false, fmt.Errorf("Error getting confirmations for %x: %v", coin.ID(), err)
+		}
+		return confs >= uint32(dc.cfg.RegFeeConfirms), nil
+	}
```

Comment:
Just noticed a misspelling on https://github.com/decred/dcrdex/pull/176/files#diff-5c4d3ee4527d6c50d1379b8edb5ef11eR533 of "request"